responsibleai/requirements-dev.txt

```diff
@@ -0,0 +1,30 @@
+# Requirements for raiwidgets development
```

Comment:
raiwidgets --> should it be responsibleai ?

Reply:
done